tools/demo/src/filters/color-overlay.js

```diff
@@ -1,9 +1,9 @@
 export default function() {
-	this.addFilter('ColorOverlayFilter', {
-		fishOnly: true,
-		args: [0xff0000],
-		oncreate(folder) {
-			folder.addColor(this, 'color');
-		}
-	});
+    this.addFilter('ColorOverlayFilter', {
+        fishOnly: true,
+        args: [0xff0000],
+        oncreate(folder) {
+            folder.addColor(this, 'color');
+        }
+    });
```

Comment:
`npm run build` did this automatically. I can remove this change from the PR if you want.

Reply:
That's fine :+1: